dashboard/test/ui/features/teacher_tools/level_summary.feature

```diff
@@ -3,6 +3,10 @@
 Feature: Level summary
 
 Scenario: Free Response level 1
+  # Turn on cfu pin and hide until it is on by default.
+  Given I am on "http://studio.code.org/test_dcdo"
```

Comment:
Please use `http://studio.code.org` instead of `http://studio.code.org/test_dcdo`.
This URL should be used to load the page so Selenium knows which domain to set the cookie for.
```suggestion
  Given I am on "http://studio.code.org"
```